### DISCUSSION TITLE
Something you might wanna listen to.

### DISCUSSION THREAD

**Discussion by DeceptedPar** [MrMeowzz/Funkin-MrMeowzz#19] [Ideas]
https://drive.google.com/file/d/1jCvEqivAJqxoJMv-MuOAonu13Y_oh8GZ/view?usp=sharing
have a listen to this, a new rendition of the "FrogIntro" music I just made
you can use this, that's why I put this inside of ideas.

**Comment by DeceptedPar** [comment]
it starts off normal at the beggining

**Reply by MrMeowzz** [reply]
I need access

**Reply by DeceptedPar** [reply]
Fixed it

**Reply by DeceptedPar** [reply]
sorry 'bout that!

**Reply by DeceptedPar** [reply]
it changes when it "kicks in"

**Reply by DeceptedPar** [reply]
I also tried to make it loop

**Comment by DeceptedPar** [comment]
![image](https://user-images.githubusercontent.com/64243677/127143188-384a0826-1c03-49f0-a15c-3d05867840e5.png)

**Reply by DeceptedPar** [reply]
yeah, I put alot of effort into something that was/is mainly a joke thing

**Reply by DeceptedPar** [reply]
reuploading it cause I changed something

**Comment by DeceptedPar** [comment]
![image](https://user-images.githubusercontent.com/64243677/127148181-73fc5b85-ed9e-490f-bc4d-a136c22ca220.png)

**Reply by DeceptedPar** [reply]
same link : https://drive.google.com/file/d/1jCvEqivAJqxoJMv-MuOAonu13Y_oh8GZ/view?usp=sharing

**Reply by DeceptedPar** [reply]
if you want me to remove anything you can tell me

**Reply by MrMeowzz** [reply]
No it's good

**Reply by DeceptedPar** [reply]
alright

**Reply by DeceptedPar** [reply]
dababy frog
